src/test/java/com/algorand/algosdk/integration/Stepdefs.java

```diff
@@ -989,14 +832,9 @@ public void checkMicro(long ma) {
         assertThat(microalgos).isEqualTo(BigInteger.valueOf(ma));
     }
 
-    @Then("I get account information")
-    public void accInfo() throws ApiException {
-        acl.accountInformation(addresses.get(0));
-    }
-
     @Then("I can get account information")
-    public void newAccInfo() throws ApiException, NoSuchAlgorithmException, com.algorand.algosdk.kmd.client.ApiException {
-        acl.accountInformation(pk.encodeAsString());
+    public void newAccInfo() throws Exception {
```

Comment:
Is it okay to use the general `Exception` here instead of specific exception instances?

Reply:
I think it is because of the `execute()` method in `aclv2` client, that throws a general `Exception`.

Reply:
Ah interesting, it looks like it's due to some code here: algorand/java-algorand-sdk/src/main/java/com/algorand/algosdk/v2/client/common/Client.java

It looks like the client dependency uses an [`IOException`](https://square.github.io/okhttp/2.x/okhttp/com/squareup/okhttp/Call.html#execute--) but we previously declared it to be an `Exception` instead. 

I think this is outside the scope of this PR anyway, so I'll close this.